tests/acceptance/stepDefinitions/provisioningContext.js

```diff
@@ -67,12 +68,12 @@ function createUser (userId, password, displayName = false, email = false) {
 function deleteUser (userId) {
   const headers = httpHelper.createAuthHeader(client.globals.backend_admin_username)
   userSettings.deleteUserFromCreatedUsersList(userId)
-  return fetch(client.globals.backend_url + '/ocs/v2.php/cloud/users/' + userId, { method: 'DELETE', headers: headers })
+  return fetch(backendHelper.getCurrentBackendUrl() + '/ocs/v2.php/cloud/users/' + userId, { method: 'DELETE', headers: headers })
```

Comment:
It would be better if the line was shorter.Can you please split into two?

Reply:
Also, use `join()` as it takes care of removing/appending slash in the between.